package.json

```diff
@@ -46,8 +46,8 @@
     "version": "changeset version && node scripts/versionPlugin.js",
     "version:status": "changeset status",
     "release": "npm run build && changeset publish",
-    "lint": "eslint ./packages --ext js,jsx,ts,tsx --max-warnings=0 --config .eslintrc.js",
-    "lint:fix": "eslint ./packages --ext js,jsx,ts,tsx --max-warnings=0 --config .eslintrc.js --fix"
```

Comment:
This change enables individual packages to override the base eslint rules.

In the case of this pull request, it is appropriate to allow the usage console.log() `@faustwp/cli` (node environment), but not within `@faustwp/core` (browser)